lstchain/calib/camera/calibration_calculator.py

```diff
@@ -0,0 +1,258 @@
+
+    pedestal_calculator: lstchain.calib.camera.pedestal
+         The pedestal to use. If None, then
+           PedestalCalculator will be used by default.
+
+    kwargs
+
+    """
+
+    pedestal_product = traits.enum_trait(
+        PedestalCalculator,
+        default='PedestalIntegrator'
+    )
+
+    flatfield_product = traits.enum_trait(
+        FlatFieldCalculator,
+        default='FlasherFlatFieldCalculator'
+    )
+
+    classes = List([
+                    FlatFieldCalculator,
+                    PedestalCalculator
+                    ]
+                   + traits.classes_with_traits(FlatFieldCalculator)
+                   + traits.classes_with_traits(PedestalCalculator)
+
+                   )
+
+    def __init__(
+        self,
+        **kwargs
+
+    ):
+
+        """
+        Parent class for the camera calibration calculators.
+        Fills the MonitoringCameraContainer on the base of calibration events
+
+        Parameters
+        ----------
+
+        flatfield_calculator: lstchain.calib.camera.flatfield
+             The flatfield to use. If None, then FlatFieldCalculator
+                will be used by default.
+
+        pedestal_calculator: lstchain.calib.camera.pedestal
+             The pedestal to use. If None, then
+               PedestalCalculator will be used by default.
+
+        kwargs
+
+        """
+
+        super().__init__(**kwargs)
+
+        self.flatfield = FlatFieldCalculator.from_name(
+            self.flatfield_product,
+            **kwargs
+        )
+        self.pedestal = PedestalCalculator.from_name(
+            self.pedestal_product,
+            **kwargs
+        )
+
+        msg = "tel_id not the same for all calibration components"
+        assert self.pedestal.tel_id == self.flatfield.tel_id, msg
+
+        self.tel_id = self.flatfield.tel_id
+
+        self.log.debug(f"{self.pedestal}")
+        self.log.debug(f"{self.flatfield}")
+
+
+class LSTCalibrationCalculator(CalibrationCalculator):
+    """
+    Calibration calculator for LST camera
+    Fills the MonitoringCameraContainer on the base of calibration events
+
+    Parameters:
+    ----------
+    minimum_hg_charge_median :
+              Temporary cut on HG charge till the calibox TIB do not work
+             (default for filter 5.2)
+
+    maximum_lg_charge_std
+             Temporary cut on LG std against Lidar events till the calibox TIB do not work
+            (default for filter 5.2)
+
+    time_calibration_path:
+            Path with the drs4 time calibration corrections
+    """
+
+    minimum_hg_charge_median = Float(
+        5000,
+        help='Temporary cut on HG charge till the calibox TIB do not work (default for filter 5.2)'
+    ).tag(config=True)
+
+    maximum_lg_charge_std = Float(
+        300,
+        help='Temporary cut on LG std against Lidar events till the calibox TIB do not work (default for filter 5.2) '
+    ).tag(config=True)
+
+    def __init__(self, **kwargs):
```

Comment:
You don't need this __init__ method if it just calls `super.__init__`, can be removed.